DESCRIPTION

```diff
@@ -38,15 +22,17 @@ Imports:
     tidyr,
     Rcpp,
     sf,
-    udpipe,
-    tidytext,
-    textdata
+    attempt,
+    DT,
+    glue,
+    openxlsx,
+    markdown
```

Comment:
Don't need these?

Reply:
I used openxls for reading income deprivation excel file.

markdown was added here as when i run dev_run.R file, it threw error not having markdown package.